### PR TITLE
Doc Android Biometric Automated Testing RDC

### PR DESCRIPTION
**Pull request by sweeneyskirt-sl**
### Description
Updated the Biometric Auth --> Automated --> Android Real Device section.

### Motivation and Context
Android Real Device testing is now supported for biometric auth.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by sweeneyskirt-sl** [comment]
Hello @lauramthomas, @linafarid , & @kristofmuhi -- I took a stab at updating the docs for this section. I actually combined the iOS and Android sections in Automated Testing section because I *think* they use the same capability and commands, but I am not 100% sure, so feel free to comment if they need to be separated. I added tabs to show separate code samples for each. I am going to put the PR in draft mode until we have a demo code sample we can reference and I am handing back to Laura to manage. Hope this helps!

File: docs/mobile-apps/features.md [hunk 1]
```diff
@@ -48,12 +43,11 @@ To initiate a fingerprint or facial recognition action in your live test session
 
 See [Live Mobile App Testing Toolbar](/mobile-apps/live-testing/live-mobile-app-testing/#app-settings) for more information.
 
-
 ### Automated Testing
 
-For automated testing, biometric authentication is supported for simulators, emulators, and iOS real devices. Please note that not all iOS and Android devices support Touch ID or Face ID. Be prepared to handle test cases..
+For automated testing, biometric authentication is supported for simulators, emulators, and iOS and Android real devices. Please note that not all iOS and Android devices support fingerprint or racial recognition, so you may have to adjust your test cases accordingly.
```
Comment (linafarid) [inline docs/mobile-apps/features.md:48]: Just to be sure, are we using fingerprint and face recognition as generic terms here or are we referring to the exact nomenclature of Android and iOS.
If the latter then we need to specific everywhere:
Android -> face recognition, fingerprint
iOS -> Face ID and Touch ID

File: docs/mobile-apps/features.md [hunk 2]
```diff
@@ -132,21 +125,41 @@ Try running one of the below example scripts, which demonstrate the default capa
   ```
   </details>
 
-<br/>
 
-#### **iOS Real Devices**
+#### Real Devices
+
+To enable Touch ID and Face ID on iOS and Android real devices, add the `allowTouchIdEnroll` capability in your test script and set it to `true`.
```
Comment (linafarid) [inline docs/mobile-apps/features.md:131]: same here as above. We should be consistent.

File: docs/mobile-apps/features.md [hunk 3]
```diff
@@ -132,21 +125,41 @@ Try running one of the below example scripts, which demonstrate the default capa
...
+
+When your test triggers a biometric prompt event, you can trigger a successful or failed authentication using the following commands:
 
-To enable Touch ID and Face ID on iOS real devices, add the `allowTouchIdEnroll` capability in your test script and set it to true, as shown in the example below:
```
Comment (linafarid) [inline docs/mobile-apps/features.md:139]: Do we not need to mention the "allowTouchIdEnroll" here anymore?

**Review by linafarid — COMMENTED** [review]
thanks @sweeneyskirt-sl @lauramthomas I have added a few comments for consistency.

File: docs/mobile-apps/features.md (hunk 1 above)
Comment (sweeneyskirt-sl) [reply docs/mobile-apps/features.md:48]: Thanks, @linafarid . I just saw they typo there....guessing "racial" recognition is not a think and might get me in trouble! Eek!

Comment (sweeneyskirt-sl) [reply docs/mobile-apps/features.md:48]: I will fix that, but I will defer to you and the experts as to whether we are using generic or specific terms.

Comment (sweeneyskirt-sl) [reply docs/mobile-apps/features.md:48]:
```suggestion
For automated testing, biometric authentication is supported for simulators, emulators, and iOS and Android real devices. Please note that not all iOS and Android devices support fingerprint or facial recognition, so you may have to adjust your test cases accordingly.
```

File: docs/mobile-apps/features.md (hunk 3 above)
Comment (sweeneyskirt-sl) [reply docs/mobile-apps/features.md:139]: It is my understanding from the sample code that you need both -- you have to set the `allowTouchIdCEnroll` capability in order to be able to execute the commands. @wswebcreation -- perhaps you can provide some insight on the exact requirements and whether they are different for iOS vs. Android. Thanks!

**Comment by sweeneyskirt-sl** [comment]
This PR is no longer viable because the structure of the doc has changed. Closing and will create a new PR

**Pull request closed without merging**